src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -940,6 +940,27 @@ public MultiCallResponse multiCall( MultiCallRequest multiCallRequest, KojiSessi
         return ret;
     }
 
+    /**
+     * Generic multiCall method. This is for homogeneous request, i.e., the "method" in multicall request objects are same.
+     * Users do not need to construct their own multicall request object.
+     * @param method
+     * @param session
+     * @param args arguments list. Each element is for one KojiMultiCallObj.
+     *             If each call object has more than one parameters, use List or array for the element type.
+     * @param type result object type
+     * @param <T>
+     * @return a list containing objects of type T
+     */
+    public <T> List<T> multiCall( String method, KojiSessionInfo session, List<Object> args, Class<T> type )
+    {
```

Comment:
I think in other methods, the convention is to always have KojiSessionInfo be the last parameter.

Reply:
+1

Reply:
amended.